clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -89,6 +90,12 @@ type Controller struct {
 	templateMapping map[string]interface{}
 	// diskMetadata store name and zone of Prometheus persistent disk.
 	diskMetadata prometheusDiskMetadata
+	// snapshotted is a check if the Prometheus snapshot is already done
+	snapshotted bool
+	// snapshotError contains error from snapshot attempt
+	snapshotError error
+	// snapshotLock makes sure that only single Prometheus snapshot is happening
+	snapshotLock sync.RWMutex
```

Comment:
s/RWMutex/Mutex/

Also - move mutex above the fields it is protecting

Reply:
Done.